zookeeper-server/src/test/resources/test-scripts.sh

```diff
@@ -217,6 +217,9 @@ stop
 
 $ZKSI --force --myid=1 --configfile "$ZOOCFGDIR/$ZOOCFG" || fail $LINENO
 
+#test version script
```

Comment:
is this file working on maven build ? 
It checks for "ls build/zookeeper*.jar" but the mavenized build does not create such file.

If I am not wrong, do you mind @szepet fixing it up ? otherwise as soon as we drop the ant-based build this script will need a fix.
Honestly I didn't known such file existed

Reply:
Is this script actually used anywhere in our code?
How is it supposed to work?